opencraft/settings.py

```diff
@@ -45,7 +46,7 @@
 # Keep the secret key used in production secret
 SECRET_KEY = env('SECRET_KEY')
 
-ALLOWED_HOSTS = env.json('ALLOWED_HOSTS', default='[]')
+ALLOWED_HOSTS = json.loads(env.json('ALLOWED_HOSTS', default='[]'))
```

Comment:
@antoviaque You'd think that it would, and the [docs](https://django-environ.readthedocs.io/en/latest/#environ.environ.Env.json) are terse; but I double-checked this in the shell and found that it does indeed return a string:

```
In [9]: import environ
In [11]: env = environ.Env()
In [12]: bar = env.json('BAR', default='[]')
In [13]: bar
Out[13]: '[]'
In [14]: type(bar)
Out[14]: str
```